includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1048,6 +1048,19 @@ public function create_intent( $order, $prepared_source ) {
 			$request['customer'] = $prepared_source->customer;
 		}
 
+		return $request;
```

Comment:
Let's add a filter here so that new parameters can be added by other extensions / code.

Discussion here:

https://github.com/woocommerce/woocommerce-gateway-stripe/pull/970#issuecomment-524338359

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/962/commits/ea36ae815ede873af7244a4927db456e43e6b38f